docker/etna-apache/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM library/httpd:2.4.43
```

Comment:
Base apache image used for the two types of apache services.  I need to provide some additional documentation on my propose development / production networking approach.